src/shell/services/instance.ts

```diff
@@ -116,16 +116,18 @@ export const instanceApi = createApi({
       keepUnusedDataFor: 0.0001,
     }),
     searchContent: builder.query<ContentItem[], SearchQuery>({
-      query: ({ query, limit, ...restOpts }) => {
-        const params = new URLSearchParams({
+      query: ({ query, ...rest }) => ({
+        url: `search/items`,
+        params: {
           q: query,
-          ...restOpts,
-        });
-        if (limit) params.set("limit", limit.toString());
-        return `search/items?${params.toString()}`;
-      },
+          ...rest,
+        },
+      }),
       transformResponse: getResponseData,
-      providesTags: ["SearchQuery"],
+      // Makes sure that the query is ran everytime the user searches.
+      // Prevents issue where the query no longer hits the endpoint
+      // when the case-sensitive value on the query parameter is the same as a previous query
+      keepUnusedDataFor: 0.0001,
```

Comment:
So what happens is if I originally searched for search/items?q=test, then again do search/items?q=test it doesn't re-query. If the case has changed then it will, but using that same casing again will not trigger a re-query.